docs/quick-start.md

```diff
@@ -389,7 +389,7 @@ As an example, we'll create a simple Spark application, `SimpleApp.py`:
 from pyspark.sql import SparkSession
 
 logFile = "YOUR_SPARK_HOME/README.md"  # Should be some file on your system
-spark = SparkSession.builder().appName(appName).master(master).getOrCreate()
+spark = SparkSession.builder.appName("SimpleApp").getOrCreate() 
```

Comment:
I guess that should be fine.

<img width="450" src="https://user-images.githubusercontent.com/6477701/28550607-12f30adc-711d-11e7-93b5-9fe07738b78a.png">

it is described below and this looks doing the same thing with equivalent Scala / Java examples here.